src/routes/docs/tutorials/sveltekit-ssr-auth/step-3/+page.markdoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: tutorial
+title: Initialize SDK
+description: Add Authentication to a SvelteKit project using Appwrite.
+step: 3
+---
+Before you can use Appwrite, you need to create the Appwrite `Client` and set the project ID and endpoint. 
+The client is then used to create services like `Databases` and `Account`, so they all point to the same Appwrite project.
+
+Create a function to build services you need in a file like `src/lib/server/appwrite.js` and **exporting the instances**.
+
+As part of the function, set the current user's session if they are logged in. This is done by accessing the session cookie from the request and calling the `setSession(session)` with the cookie value.
+
+```js
+// src/lib/server/appwrite.js
+import { Client, Account } from 'node-appwrite';
+import { APPWRITE_KEY } from '$env/static/private';
+import { PUBLIC_APPWRITE_ENDPOINT, PUBLIC_APPWRITE_PROJECT_ID } from '$env/static/public';
+
+export const SESSION_COOKIE = 'my-custom-session';
+
+export function createAppwriteClient(event) {
+	// Build the Appwrite client
+	const client = new Client()
+		.setEndpoint(PUBLIC_APPWRITE_ENDPOINT)
+		.setProject(PUBLIC_APPWRITE_PROJECT_ID);
+
+	// Set the API key for the client
+	client.setKey(APPWRITE_KEY);
+
+	// Extract our custom domain's session cookie from the request
+	const session = event.cookies.get(SESSION_COOKIE);
+	if (session) client.setSession(session);
+
+	// Return the services we want to use.
+  // This allows us to use syntax like:
+  // import { createAppwriteClient } from '$lib/server/appwrite';
+  //
+  // const { account } = createAppwriteClient(event);
+  // const user = await account.get();
+	return {
+		get account() {
+			return new Account(client);
+		}
+	};
+}
+```
+
+`APPWRITE_KEY`, `PUBLIC_APPWRITE_ENDPOINT` and `PUBLIC_APPWRITE_PROJECT` are environment variables that are exported in your project's [.env file](https://kit.svelte.dev/docs/modules#$env-dynamic-public).
+
+You can get the values for these variables from the Appwrite console. The `PUBLIC_APPWRITE_ENDPOINT` and `PUBLIC_APPWRITE_PROJECT` are the endpoint and project ID for your Appwrite project. 
+
+For example, your `.env` might look something similar to this.
+
+```text
+APPWRITE_KEY=4d5e6f7g8h9i0j1k2l3m4n5o
```

Comment:
@loks0n Hey can you make sure these are <PLACE_HOLDER> type tags instead? No real IDs please

Reply:
Done